autospec/avxjudge.py

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/python3
+import subprocess
+import sys
+import re
+import argparse
+import os
+
+# MMX and SSE2 instructions
+sse_instructions_xmm = list()
+
+# 0.1 value instructions
+avx2_instructions_lv = list()
```

Comment:
Just for your own edification, not a big deal here, prefer using the literal object instead of the constructor. The call to `list()` or `dict()` here causes an extra lookup for that function while the interpreter immediately knows what to do with `[]` or `{}`... not a big deal though.